python_hll/serialization.py

```diff
@@ -644,7 +644,7 @@ def pack_cutoff_byte(cls, explicit_cutoff, sparse_enabled):
         :rtype: byte
         """
         sparse_bit = BitUtil.left_shift_int(1, cls.EXPLICIT_CUTOFF_BITS) if sparse_enabled else 0
-        return BitUtil.to_signed_byte(sparse_bit | (cls.EXPLICIT_CUTOFF_MASK & explicit_cutoff))
+        return sparse_bit | (cls.EXPLICIT_CUTOFF_MASK & explicit_cutoff)
```

Comment:
The corresponding Java code explicitly casts to a byte, which is why we do it here:

https://github.com/aggregateknowledge/java-hll/blob/1f4126e79a85b79581460c75bf1c1634b8a7402d/src/main/java/net/agkn/hll/serialization/SerializationUtil.java#L187